one-on-ones.md

```diff
@@ -0,0 +1,25 @@
+# Project 2 One-on-One Objectives
+
+## Friday (15 minutes)
+- Communicate and lingering concerns and/or confusion about Unit 2 topics.
+- Identify at least one topic to review or research over the weekend in preparation for Project 2
+- Discuss possible project ideas and identify strong candidates
```

Comment:
I like all three of these, but it's a lot for 15 minutes, I suspect we should maybe just do the last 2 instead? Save Unit 2 questions for review sessions?